hedera-mirror-rest/api/v1/openapi.yml

```
@@ -1923,6 +1923,22 @@ components:
           type: string
         service_endpoints:
           $ref: '#/components/schemas/ServiceEndpoints'
+        stake:
+          description: The node consensus weight at the beginning of the staking period
+          nullable: true
+          type: number
+        stake_rewarded:
+          description: The sum (balance + staked) for all accounts staked to the node that are not declining rewards at the beginning of the staking period
+          nullable: true
+          type: number
+        stake_total:
+          description: The sum of all nodes' consensus weight at the beginning of the staking period
+          nullable: true
+          type: number
+        staking_period:
+          description: The timestamps at the beginning of the staking period
+          nullable: true
+          $ref: '#/components/schemas/TimestampRange'
```

openapi won't render it as you expected, to customize an existing schema, you need to use allOf
```suggestion
         allOf:
           - $ref: '#/components/schemas/TimestampRange'
           - description: The timestamps at the beginning of the staking period
              nullable: true
```

The properties of allOf[n] will get merged into the properties of allOf[0..n-1]